cluster-install/install.sh

```diff
@@ -1129,6 +1132,20 @@ wait_for_deployment_running() {
     [ -z "$deployment" ] && fatal "no deployment name given"
 
     loginfo "Waiting on deployment/$deployment (timeout: $timeout)..."
+
+    # Wait for the deployment to be created. The subsequent 'rollout status' command will fail if called before the
+    # deployment has been created.
+    max_retries=15
```

Comment:
I was seeing my connector deployment show up after this check had already failed. Not sure why the deployment was taking so long to appear but adding this check to first wait for the deployment before calling the 'rollout status' command below fixed it